bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/NetatmoModuleHandler.java

```diff
@@ -34,6 +34,8 @@
  * module device accessed through the Netatmo Device
  *
  * @author Gaël L'hopital - Initial contribution
+ * @author Rob Nielsen - Added day, week, and month measurements to the weather station and modules
+ *
```

Comment:
You can remove this as well, there are no other modifications to this file

Reply:
done